docs/Using-Jupyter-Notebooks.md

```diff
@@ -0,0 +1,89 @@
+# Using Jupyter Notebooks for Animal-AI
```

Comment:
For me this file would come under the same umbrella as the `Background-*` files. Jupyter notebooks are really well documented elsewhere and I don't think we should attempt to replicate that here

Reply:
I dont think we should send the user to official documentation all the time. We should provide them with the basics and given this is an area that some of us struggled with (myself including, as a newcomer), i wanted to negate the confusion for the user and have this in here. This doc really is a light read and will require little maintenance.